snuba/cli/migrations.py

```diff
@@ -130,3 +135,93 @@ def reverse(
         raise click.ClickException(str(e))
 
     click.echo(f"Finished reversing migration {migration_key}")
+
+
+@migrations.command()
+@click.option(
+    "--type", "node_type", type=click.Choice(["local", "dist"]), required=True
+)
+@click.option(
+    "--storage-set",
+    "storage_set_names",
+    type=click.Choice([s.value for s in StorageSetKey]),
+    required=True,
+    multiple=True,
+)
+@click.option(
+    "--host-name",
+    type=str,
+    required=True,
+    default=os.environ.get("CLICKHOUSE_HOST", "localhost"),
+)
+@click.option(
+    "--port",
+    type=int,
+    required=True,
+    default=int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+)
+@click.option(
+    "--user",
+    type=str,
+    required=True,
+    default=os.environ.get("CLICKHOUSE_USER", "default"),
+)
+@click.option(
+    "--password",
+    type=str,
+    required=True,
+    default=os.environ.get("CLICKHOUSE_PASSWORD", ""),
+)
```

Comment:
Getting them from the environment is better. Please remove the parameters entirely.

Reply:
deleted